Sources/FaunaDB/Query.swift

```diff
@@ -55,6 +55,23 @@ public struct Var: Fn {
     }
 }
 
+/// `At` evaluates the expr at a given timestamp.
+///
+/// [Reference](https://fauna.com/documentation/queries#basic_forms).
+public struct At: Fn {
+
+    var call: Fn.Call
+
+    /// - Parameter timestamp: A timestamp in which the expr will be evaluated.
+    /// - Parameter expr:      The expression to be evaluated.
+    public init(_ timestamp: Expr, _ expr: Expr) {
```

Comment:
I think we should keep the first parameter named. It feels more idiomatic for Swift.